File: app/src/main/java/ru/deniskrd/android/webbrowser/db/entities/Favourite.java
package ru.deniskrd.android.webbrowser.db.entities;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

@Entity(tableName = "favourites")
@NoArgsConstructor
@AllArgsConstructor
public class Favourite {

    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "name")
    private String name;

    @ColumnInfo(name = "url")
    private String url;

    public String getName() {
        return name;
    }

    public String getUrl() {
        return url;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: app/src/main/java/ru/deniskrd/android/webbrowser/db/entities/Favourite.kt
package ru.deniskrd.android.webbrowser.db.entities

import androidx.annotation.NonNull
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "favourites")
data class Favourite (

    @PrimaryKey
    @NonNull
    val name : String = "",

    val url : String? = null
)
